fedimint-server/src/net/peers.rs

```diff
@@ -81,12 +81,47 @@ struct PeerConnectionStateMachine<M> {
     state: PeerConnectionState<M>,
 }
 
+/// Calculates delays for reconnecting to peers
+/// The default values are in the order of seconds
+#[derive(Debug, Clone, Copy)]
```

Comment:
`DelayCalculator` has a name making it seems like a code-object. But it's really just a piece of data. `DelaySettings` maybe or something?
But even then ... it only has `scaling_factor` and `reconnection_delay` method. So maybe `ConnectionSettings` or something? Not sure. Just saying that seems a bit off. I'm big on maintaining clear distinction between "code" and "data".

Reply:
Not a big deal one way or another.